src/tools/lifter/selftest.sml

```diff
@@ -467,12 +472,17 @@ in () end;
 
 fun m0_lift_instr mu_b mu_e pc hex_code desc = let
   val r1 = (print "LP "; test_M0_1.lift_instr mu_b mu_e pc hex_code desc)
-in if (test_fast) then (r1, r1, r1, r1) else let
+in if (test_fast) then (r1, r1, r1, r1, r1, r1, r1, r1) else let
```

Comment:
Shouldn't you include at least one of `M0_mod` into the fast tests? (I don't know how big are the changes)

Reply:
in this case I guess it should be ok without for fast testing. but this is up to the meaning of fast testing of course.